public/app/features/users/UsersTable.test.tsx

```diff
@@ -15,39 +15,45 @@ jest.mock('app/core/core', () => ({
   },
 }));
 
-const setup = (propOverrides?: object) => {
+function getProps(propOverrides?: object) {
   const props: Props = {
     users: [] as OrgUser[],
     onRoleChange: jest.fn(),
     onRemoveUser: jest.fn(),
   };
 
-  Object.assign(props, propOverrides);
-
-  return shallow(<UsersTable {...props} />);
-};
+  return Object.assign(props, propOverrides);
+}
 
 describe('Render', () => {
   it('should render component', () => {
-    const wrapper = setup();
-
-    expect(wrapper).toMatchSnapshot();
+    render(<UsersTable {...getProps()} />);
```

Comment:
this test doesn't have an assertion. try to make sure your tests always contain an `expect` 
```suggestion
    expect(() => render(<UsersTable {...getProps()} />)).not.toThrow();
```